synapse/rest/client/v1/login.py

```diff
@@ -67,6 +77,16 @@ def on_POST(self, request):
                     "uri": "%s%s" % (self.idp_redirect_url, relay_state)
                 }
                 defer.returnValue((200, result))
+            elif self.cas_enabled and (login_submission["type"] ==
+                                       LoginRestServlet.CAS_TYPE):
+                url = "%s/proxyValidate" % (self.cas_server_url)
```

Comment:
FYI, there should be a comma after `self.cas_server_url` to make it a tuple, otherwise the brackets don't actually do anything here.